Gems/Atom/Tools/MaterialCanvas/Assets/MaterialCanvas/GraphData/Nodes/Functions/trunc.materialgraphnode

```diff
@@ -5,8 +5,9 @@
     "ClassData": {
         "id": "{26129B8D-5E3D-4843-B0BD-2017F2CD5AED}",
         "category": "Math Functions",
-        "title": "Trunc",
+        "title": "Truncate",
         "titlePaletteName": "MathNodeTitlePalette",
+        "description": "Truncates floating-point value to integer value",
```

Comment:
What's the difference between this and Floor?

Reply:
Interesting. I based the description off of the HLSL documentation.